Tests/BoundariesTests/BoundariesTests.swift

```diff
@@ -33,81 +37,3 @@ final class BoundariesTests: XCTestCase {
   }
 }
 
-func print<A>(_ string: String) -> Effect<A> {
```

Comment:
moved all this to a separate test file so that we could easily see it one place.